xarray/tests/test_plugins.py

```diff
@@ -1,10 +1,19 @@
 from unittest import mock
 
-import pkg_resources
 import pytest
 
 from xarray.backends import common, plugins
 
+try:
+    from importlib.metadata import EntryPoint
+
+    importlib_metadata_mock = "importlib.metadata"
+except ImportError:
+    # if the fallback library is missing, we are doomed.
+    from importlib_metadata import EntryPoint  # type: ignore[no-redef]
+
+    importlib_metadata_mock = "importlib_metadata"
```

Comment:
```suggestion
if sys.version_info >= (3, 8):
    from importlib.metadata import EntryPoint

    importlib_metadata_mock = "importlib.metadata"
else:
    # if the fallback library is missing, we are doomed.
    from importlib_metadata import EntryPoint

    importlib_metadata_mock = "importlib_metadata"
```

Have to import sys as well.

Reply:
Thanks @Illviljan, I've just seen your PR #5988.